plugins/woocommerce-admin/client/subscriptions/style.scss

```diff
@@ -1,35 +1,109 @@
-.wcpay-empty-subscriptions__container {
-	display: flex;
-	flex-direction: column;
-	justify-content: center;
-	align-items: center;
-	padding: 48px 0;
-	position: static;
-	height: 344px;
-	left: calc(50% - 1240px / 2);
-	top: 271px;
-	flex: none;
-	order: 1;
-	flex-grow: 0;
-	margin: 0;
-	text-align: center;
-
-	.wcpay-empty-subscriptions__description {
-		max-width: 535px;
-		margin: 15px 0;
+$max-width: 680px;
+
+.wcpay-empty-subscriptions__error {
+	margin: 0 auto 24px;
+	max-width: $max-width;
+}
+
+.subscriptions-page {
+	margin: 0 auto;
+	max-width: $max-width;
+
+	.components-card {
+		margin-bottom: 24px;
+	}
+
+	.wcpay-empty-subscriptions-page-payment-methods {
+		svg {
+			display: inline-block;
+			height: 35px;
+			width: 52px;
+			margin-right: 0.5em;
+			margin-bottom: 0.5em;
+		}
+	}
+
+	.wcpay-empty-subscriptions-page-terms-of-service {
+		display: block;
+		margin: 0.5em 0;
 	}
 
-	.wcpay-empty-subscriptions__tos {
-		font-size: 11px;
-		color: #757575;
-		margin-bottom: 6px;
+	.subscriptions-page-onboarding-steps {
+		@include breakpoint( '>600px' ) {
+			display: flex;
+			flex-direction: row;
+		}
+
+		.subscriptions-page-onboarding-steps-item {
+			margin-top: 30px;
+
+			@include breakpoint( '>600px' ) {
+				margin-top: 0;
+				margin-right: 80px;
+			}
+		}
 	}
 
-	.wcpay-empty-subscriptions__button_container {
-		margin: 8px 0;
+	.wcpay-empty-subscriptions-page-step-number {
+		background-color: $studio-woocommerce-purple-50;
+		color: $white;
+		font-size: 20px;
+		border-radius: 50%;
+		min-height: 32px;
+		min-width: 32px;
+		display: inline-block;
+		text-align: center;
+		line-height: 32px;
 	}
 
-	.wcpay-empty-subscriptions__error {
-		margin: 15px 0;
+	.subscriptions {
+		h2 {
+			font-size: 20px;
+			font-weight: 300;
```

Comment:
tiny nitpick here but I believe a font weight of 400 matches the design and is consistant with headings on other pages like the WooCommerce > Home screen.

Reply:
Thanks James! Fixed in f34fcab